python/paddle/jit/dy2static/function_spec.py

```diff
@@ -194,6 +194,26 @@ def pir_to_static_inputs_with_spec(self, input_with_spec, main_program):
                         dtype=convert_dtype(var_spec.dtype),
                     )
                     feed_value.stop_gradient = stop_gradient
+
+                    # warp dist tensor
+                    from paddle.distributed.auto_parallel.static.dist_input_spec import (
+                        DistributedInputSpec,
+                    )
+
+                    if isinstance(var_spec, DistributedInputSpec):
+                        dist_dense_tensor_type = paddle.base.libpaddle.pir.create_dist_dense_tensor_type_by_dense_tensor(
```

Comment:
to change to use shard_tensor python API in future when this API is adapted for PIR. @hitywt

Reply:
agree